src/Laravel/Commands/PestTestCommand.php

```diff
@@ -39,7 +39,7 @@ public function handle(): void
         TestSuite::getInstance(base_path(), $this->option('test-directory'));
 
         /** @var string $name */
-        $name = $this->argument('name');
+        $name = Str::finish($this->argument('name'), 'Test');
```

Comment:
Laravel has this feature too? On the `make:test` command?

Reply:
Looks like Laravel doesn't have this feature either. It might be useful to have it as the default PHPUnit configuration doesn't run tests that don't have the `Test.php` suffix. When generating a new test, a developer may expect it to be loaded automatically and could be puzzling if it doesn't

Reply:
Doesn't look like it actually. 🤔
https://github.com/laravel/framework/blob/8.x/src/Illuminate/Foundation/Console/TestMakeCommand.php#L65

Reply:
I actually remove that "Test.php" suffix in my apps.

Reply:
😮 🤯

Reply:
@nunomaduro mmm I see, it's probably better to simply respect the name that developers give to their tests as this PR wouldn't cover all the use-cases 👍

Reply:
Thank you for the suggestion tho.